client/my-sites/sidebar-unified/style.scss

```diff
@@ -139,6 +144,78 @@ $font-size: rem( 14px );
 	.site__badge {
 		background: var( --color-sidebar-text );
 	}
+
+	// client/blocks/upsell-nudge/style.scss
+	.upsell-nudge.banner.card.is-compact {
+		background-color: #fff;
+		color: #000;
+	}
+
+	.upsell-nudge.banner.card.is-compact .banner__info .banner__title {
+		color: #000;
+	}
+
+	.is-sidebar-collapsed & {
+		.dashicons-admin-collapse::before {
+			transform: rotate( 180deg );
+		}
+
+		.current-site__switch-sites .button.is-borderless {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			padding: 0;
+			height: 31px;
+		}
+
+		.current-site__switch-sites .button.is-borderless .gridicon {
+			width: 14px;
+			height: 14px;
+			position: static;
+			left: inherit;
+			top: inherit;
+			margin-right: 0;
+		}
+
+		.current-site__switch-sites-label {
+			display: none;
+		}
+
+		.current-site .site .site__info {
+			display: none;
+		}
+
+		.site__content {
+			padding: 9px 6px;
+		}
+
+		.site__home {
+			top: 9px;
+			left: 5px;
+		}
+
+		.current-site__notices > a::before {
+			content: '\f534';
+			font-family: 'dashicons';
```

Comment:
This doesn't seem to work for me on calypso.live. The icon doesn't show up and the banner is collapsed.

![image](https://user-images.githubusercontent.com/1398304/94714332-da29fc00-0300-11eb-8970-49d12a5dfe60.png)

Reply:
I am not sure how you get this.. It seems that the icon is not loaded? can you try in local just in case?